pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -549,6 +549,25 @@ func (r *SubnetPortReconciler) CollectGarbage(ctx context.Context) error {
 			r.StatusUpdater.IncreaseDeleteSuccessTotal()
 		}
 	}
+
+	// In case of there are still some DHCP static bindings for SubnetPort CR in store
+	subnetPortCRSet := r.SubnetPortService.ListSubnetPortCRIDsFromDhcpStaticBindingStore()
```

Comment:
The set actually saves staticbinding id (which is the same as the subnetport id) from the staticbinding store.

I feel the name is confusing, how about 
subnetPortCRSet  -> nsxStaticBindingIdSet
ListSubnetPortCRIDsFromDhcpStaticBindingStore -> ListIDsFromDhcpStaticBindingStore

Reply:
thanks, changed as suggested.